src/components/InduvidualMovie/InduvidualMovie.js

```diff
@@ -2,18 +2,15 @@ import React from 'react'
 import './InduvidualMovie.css'
 
 function InduvidualMovie(props) {
```

Comment:
Just a minor typo/spelling thing - should be "individual".  If you change this, I'd be carefully change it in the classname on line 6 and any related styling using that classname too.